### PR TITLE
Parameter study

### LINKED ISSUE #28: Perform your first parameter study
Machine learning is known to be a computationally intensive process.

Based on your first data set and your first trained model, perform a first parameter study.

Decide on your own witch parameters to vary, but give an overview of the tested parameter space. in addition, think of important metrics in order to compare your different models.

Start simple and don't think too complicated, choose parameters that are easy to vary from a coding perspective.

It is more important to set up the first study than the actual variation. there will be many more in the future.

Divide your work into two parts. This issue is for coding. I am going to create a second one for documenting.

### PR DESCRIPTION
**Pull request by RicoKursidem**
resolves #28 

I'm not sure if I did the right thing. 
I run the neural network multiple times, variating the attributes: learning rate, epochs and batch size.

### CODE REVIEW

File: deep_groove_project/src/classification_dataset1/parameter_study.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+              'shuffle': True,
+              'learning_rate': 1e-7,
+              'epochs': 200}
+
+    parameter_batch_size(params, output={"cli": False, "plot": False})
+    parameter_learning_rate(params, output={"cli": False, "plot": False})
+    parameter_epochs(params, output={"cli": False, "plot": False})
+
+
+def parameter_epochs(params, output):
```
Comment (philippabele) [inline deep_groove_project/src/classification_dataset1/parameter_study.py:25]: Could you add a doc string to your functions ? This would simplify my work to review your code.

Comment (philippabele) [reply deep_groove_project/src/classification_dataset1/parameter_study.py:25]: https://www.programiz.com/python-programming/docstrings

File: deep_groove_project/src/classification_dataset1/parameter_study.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    epochs = [10, 50, 100, 200, 500, 1000]
+    for i in range(len(epochs)):
+        params["epochs"] = epochs[i]
+        values.append(cls_set1(params, output=output))
+        values[len(values) - 1]['epochs'] = epochs[i]
+
+    elapsed_time = time.time() - st
+    print('Execution time:', elapsed_time, 'seconds')
+
+    for val, i in zip(values, range(len(values))):
```
Comment (philippabele) [inline deep_groove_project/src/classification_dataset1/parameter_study.py]: You do not need to comment on each line of code, but I can see that you thinking and coding in blocks. May you add comments per block? This would increase the understanding of coding story :)

**Review by philippabele — CHANGES_REQUESTED** [review]

File: deep_groove_project/src/classification_dataset1/parameter_study.py (hunk 2 above)
Comment (RicoKursidem) [reply deep_groove_project/src/classification_dataset1/parameter_study.py]: I added the docstings and comments to the blocks.

I hope they are enough. Furthermore, I will try to clean up the code in the following days to minimize my repeating code snippets. The comments helped to find these snippets.

File: deep_groove_project/src/classification_dataset1/parameter_study.py (hunk 1 above)
Comment (RicoKursidem) [reply deep_groove_project/src/classification_dataset1/parameter_study.py:25]: Added them. Very cool feature I did not know of :)

File: deep_groove_project/src/classification_dataset1/parameter_study.py (hunk 2 above)
Comment (RicoKursidem) [reply deep_groove_project/src/classification_dataset1/parameter_study.py]: I simplified the show plot code by putting it into its own function

**Review by philippabele — APPROVED** [review]
